src/jobflow_remote/jobs/runner.py

```diff
@@ -282,12 +282,6 @@ def run(
                     self.update_batch_jobs
                 )
 
-        if complete:
-            self.advance_state(states)
-            scheduler.every(self.runner_options.delay_advance_status).seconds.do(
-                self.advance_state, states=states
-            )
-
```

Comment:
I don't understand this part of the code well enough to know why this was removed, so just commenting to make sure it was intentional :P

Reply:
Thanks for the warning. Indeed this was a bit involved. This portion of the code is already included in another `if` above, so it is correct to remove. I probably mistakenly duplicated it at some point.